aten/src/ATen/core/boxing/KernelFunction_impl.h

```diff
@@ -98,7 +103,8 @@ inline KernelFunction KernelFunction::makeFromUnboxedFunctor(std::unique_ptr<Ope
         nullptr, // no functorFactory_ because we already have the functor_
         std::move(kernelFunctor),
         &detail::make_boxed_from_unboxed_functor<KernelFunctor, AllowLegacyTypes>::call,
-        reinterpret_cast<void*>(&detail::wrap_kernel_functor_unboxed<KernelFunctor>::call)
+        reinterpret_cast<void*>(&detail::wrap_kernel_functor_unboxed<KernelFunctor>::call),
+        detail::FunctionSchemaInferer<KernelFunctor>().run()
```

Comment:
This fails to typecheck without `run()`, even though there's an implicit conversion out of FunctionSchemaInferer